### PR TITLE
retag image registry

### PR DESCRIPTION
**Pull request by Muzry**
- retag image registry

### CODE REVIEW

File: actions/java-agent/1.0/dice.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ### job 配置项
 jobs:
   java-agent-inject:
-    image: registry.cn-hangzhou.aliyuncs.com/dice/java-agent-action:20201106-706b7d19
+    image: erda-registry.cn-hangzhou.cr.aliyuncs.com/erda/java-agent-action:20201106-706b7d19
     cmd: tar -xf /resource/java-agent/spot-agent.tar.gz -C /dice
     envs:
-      JAVA_AGENT_OPTS: -javaagent:/dice/spot-agent/spot-agent.jar -javaagent:/dice/spot-agent/spot-profiler.jar
+      JAVA_AGENT_OPTS: -javaagent:/erda/spot-agent/spot-agent.jar -javaagent:/erda/spot-agent/spot-profiler.jar
```
Comment (luobily) [inline actions/java-agent/1.0/dice.yml:7]: check cmd

Comment (luobily) [reply actions/java-agent/1.0/dice.yml:7]: This directory is not exist

**Review by luobily — CHANGES_REQUESTED** [review]

**Comment by luobily** [comment]
cc @sfwn

File: dockerfiles/terminus-herd/herd.1.1.9-beta.2/deploy.sh [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-image=registry.cn-hangzhou.aliyuncs.com/terminus/terminus-herd:1.1.9-beta.2
+image=erda-registry.cn-hangzhou.cr.aliyuncs.com/erda/terminus-herd:1.1.9-beta.2
```
Comment (sfwn) [inline dockerfiles/terminus-herd/herd.1.1.9-beta.2/deploy.sh:3]: not exist

**Review by sfwn — CHANGES_REQUESTED** [review]

File: actions/buildpack-aliyun/1.0/Dockerfile [hunk 3]
```diff
@@ -10,7 +10,7 @@ RUN GOOS=linux GOARCH=amd64 go build -o /assets/run github.com/erda-project/erda
 WORKDIR actions/buildpack-aliyun/1.0
 RUN bash sync_assets.sh
 
-FROM registry.cn-hangzhou.aliyuncs.com/dice/centos:7
+FROM erda-registry.cn-hangzhou.cr.aliyuncs.com/erda/centos:7
```
Comment (sfwn) [inline actions/buildpack-aliyun/1.0/Dockerfile:13]: not exist

File: actions/buildpack-aliyun/1.0/bp/java/build/maven/Dockerfile [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ARG DEP_CACHE_IMAGE
 FROM ${DEP_CACHE_IMAGE} as cache-image
 
-FROM {{BP_DOCKER_BASE_REGISTRY}}/terminus/terminus-maven:3-jdk-8-alpine
+FROM {{BP_DOCKER_BASE_REGISTRY}}/erda/terminus-maven:3-jdk-8-alpine
```
Comment (sfwn) [inline actions/buildpack-aliyun/1.0/bp/java/build/maven/Dockerfile:4]: rebuild buildpack action image

File: actions/buildpack-aliyun/1.0/bp/java/pack/springboot-asia/Dockerfile [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # Build app image
-FROM {{BP_DOCKER_BASE_REGISTRY}}/terminus/terminus-openjdk:v1.8.0.242-asia
+FROM {{BP_DOCKER_BASE_REGISTRY}}/erda/terminus-openjdk:v1.8.0.242-asia
```
Comment (sfwn) [inline actions/buildpack-aliyun/1.0/bp/java/pack/springboot-asia/Dockerfile:2]: rebuild related action

File: actions/buildpack/1.0/bp/java/build/maven/Dockerfile [hunk 6]
```diff
@@ -1,7 +1,7 @@
 ARG DEP_CACHE_IMAGE
 FROM ${DEP_CACHE_IMAGE} as cache-image
 
-FROM {{BP_DOCKER_BASE_REGISTRY}}/terminus/terminus-maven:3-jdk-8-alpine
+FROM {{BP_DOCKER_BASE_REGISTRY}}/erda/terminus-maven:3-jdk-8-alpine
```
Comment (sfwn) [inline actions/buildpack/1.0/bp/java/build/maven/Dockerfile:4]: rebuild related action

File: actions/custom-script/1.0/Dockerfile [hunk 7]
```diff
@@ -1,4 +1,4 @@
-FROM registry.cn-hangzhou.aliyuncs.com/dice/git-golang-image:1.1
+FROM erda-registry.cn-hangzhou.cr.aliyuncs.com/erda/git-golang-image:1.1
```
Comment (sfwn) [inline actions/custom-script/1.0/Dockerfile:1]: not exist

File: actions/gitbook/1.0/comp/Dockerfile [hunk 8]
```diff
@@ -1,4 +1,4 @@
-FROM {{CENTRAL_REGISTRY}}/dice-third-party/terminus-nginx:0.2
+FROM {{CENTRAL_REGISTRY}}/erda/terminus-nginx:0.2
```
Comment (sfwn) [inline actions/gitbook/1.0/comp/Dockerfile:1]: rebuild action image.

File: actions/java-build/1.0/internal/test.sh [hunk 9]
```diff
@@ -1,3 +1,3 @@
...
\ No newline at end of file
+docker run -v /var/run/docker.sock:/var/run/docker.sock -v /root/demo-notice:/root -e ACTION_WORKDIR=/root -e ACTION_OPTIONS="-am -pl cp-notice-starter -Dmaven.test.skip=true" -e ACTION_TARGET=cp-notice-startger/target/cnooc-notice-starter-1.0-SNAPSHOT.jar -e ACTION_SERVICE=accounting-web -e GITTAR_REPO=jzdtest -e GITTAR_BRANCH=master -e DICE_OPERATOR_ID=jzd -e CENTRAL_REGISTRY=registry.cn-hangzhou.aliyuncs.com -e LOCAL_REGISTRY=registry.default.svc.cluster.local:5000 -e NEXUS_ADDR=nexus.default.svc.cluster.local:8081 -e NEXUS_USERNAME=admin -e NEXUS_PASSWORD=admin123  --entrypoint /opt/action/run erda-registry.cn-hangzhou.cr.aliyuncs.com/erda/mvn-action:v1.0
```
Comment (sfwn) [inline actions/java-build/1.0/internal/test.sh:3]: erda-registry.cn-hangzhou.cr.aliyuncs.com/erda/mvn-action:v1.0

not exist

File: actions/java/1.0/comp/openjdk/Dockerfile [hunk 10]
```diff
@@ -1,7 +1,7 @@
 # 可选: v1.8.0.242
 ARG CONTAINER_VERSION=v1.8.0.242
 
-FROM {{CENTRAL_REGISTRY}}/terminus/terminus-openjdk:$CONTAINER_VERSION
+FROM {{CENTRAL_REGISTRY}}/erda/terminus-openjdk:$CONTAINER_VERSION
```
Comment (sfwn) [inline actions/java/1.0/comp/openjdk/Dockerfile:4]: rebuild action image

File: actions/js/1.0/comp/herd/Dockerfile [hunk 11]
```diff
@@ -1,4 +1,4 @@
-FROM {{CENTRAL_REGISTRY}}/terminus/terminus-herd:1.1.8-node12
+FROM {{CENTRAL_REGISTRY}}/erda/terminus-herd:1.1.8-node12
```
Comment (sfwn) [inline actions/js/1.0/comp/herd/Dockerfile:1]: rebuild action image

File: actions/php/1.0/comp/Dockerfile [hunk 12]
```diff
@@ -1,4 +1,4 @@
-FROM {{CENTRAL_REGISTRY}}/dice-third-party/terminus-php-apache:{{PHP_VERSION}}
+FROM {{CENTRAL_REGISTRY}}/erda/terminus-php-apache:{{PHP_VERSION}}
```
Comment (sfwn) [inline actions/php/1.0/comp/Dockerfile:1]: rebuild action image

File: actions/release/1.0/comp/migration/Dockerfile [hunk 13]
```diff
@@ -1,5 +1,5 @@
 # Build app image
-FROM registry.cn-hangzhou.aliyuncs.com/dice-third-party/migration:1.0.2
+FROM erda-registry.cn-hangzhou.cr.aliyuncs.com/erda/migration:1.0.2
```
Comment (sfwn) [inline actions/release/1.0/comp/migration/Dockerfile:2]: rebuild action image

**Review by sfwn — CHANGES_REQUESTED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**